metabase/metabase
# Fix public dashboards are cut when printing

Fixes #41926

This PR fixes the cut-off of public dashboards when trying to print the page. The root cause is the `overflow: auto` style on `EmbedFrame` (which is also used for public dashboards) added to address #40660. Fixed by making public dashboards use `overflow: visible` when printing.

### To verify

1. Create a dashboard with a table dashcard that shows 50 rows
2. Create and open a public link
3. Press CMD + P / Ctrl + P to print the page
4. Ensure the print preview contains all 50 rows

### Demo

**Before**

https://github.com/user-attachments/assets/b226cbc3-69c8-4cf9-b1fb-9e5c5fd9aa48

**After**

https://github.com/user-attachments/assets/b536653e-be6d-4bf5-ac88-af3d4cd7ad24
